Sources/ArcGISToolkit/Utility/CredentialInputView.swift

```diff
@@ -210,65 +210,58 @@ struct CredentialInputSheetView: View {
-                            .foregroundColor(.secondary)
-                            .multilineTextAlignment(.center)
-                    }
-                    .padding(.vertical)
-                    VStack {
-                        switch fields {
-                        case .password:
-                            passwordTextField
-                        case .usernamePassword:
-                            usernameTextField
-                            Divider()
-                            passwordTextField
-                        }
-                        Divider()
-                    }
-                    .padding([.bottom, .horizontal])
-                    HStack {
-                        Spacer()
-                        Button(role: .cancel) {
-                            cancelAction.handler("", "")
-                        } label: {
-                            Text(cancelAction.title)
-                                .padding(.horizontal)
+        NavigationStack {
+            GeometryReader { proxy in
+                VStack {
+                    VStack(alignment: .center) {
+                        VStack(spacing: 8) {
+                            Text(title)
+                                .font(.title)
+                                .multilineTextAlignment(.center)
+                            Text(message)
+                                .font(.subheadline)
+                                .foregroundColor(.secondary)
+                                .multilineTextAlignment(.center)
                         }
-                        .buttonStyle(.bordered)
-                        Spacer()
-                        Button {
-                            isPresented.wrappedValue = false
-                            continueAction.handler(username, password)
-                        } label: {
-                            Text(continueAction.title)
-                                .padding(.horizontal)
+                        .padding(.vertical)
+                        Form {
+                            switch fields {
+                            case .password:
+                                passwordTextField
+                            case .usernamePassword:
+                                usernameTextField
+                                passwordTextField
+                            }
                         }
-                        .buttonStyle(.borderedProminent)
-                        .disabled(!isContinueEnabled)
-                        Spacer()
                     }
-                    Spacer()
                 }
-                .padding()
-                Spacer()
             }
-        }
-        .onAppear {
-            // Set initial focus of text field.
-            switch fields {
-            case .usernamePassword:
-                usernameFieldIsFocused = true
-            case .password:
-                passwordFieldIsFocused = true
+            .toolbar {
```

Comment:
The cancel and continue buttons were both oddly placed at the bottom which doesn't follow this guideline:
<img width="618" alt="Screenshot 2024-09-18 at 5 19 02 PM" src="https://github.com/user-attachments/assets/4cec41a9-38e9-4d14-9824-869576d6245f">

Reply:
Does this need to be part of the visionOS work? If not, then I'd prefer we do not do this at this time. We have a UI/UX designer starting in October and one of the things we will ask her to do is to go through this UI and make suggestions for improvements.

Reply:
> Does this need to be part of the visionOS work?

I don't think this **needs** to be. But putting the text fields into a form would need to because that adds the default background color for each text field which would be required for visionOS as per [this](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/878#discussion_r1765903581) comment. And I don't think that would change much on iOS.

Reply:
The form change can be ok. But I don't want to move the buttons or do anything else